test/validators/movie.test.js

```diff
@@ -6,6 +6,18 @@ const MovieValidator = require('../../lib/validators/movies/create');
 
 describe('movie validator', () => {
 
+  describe('name', () => {
```

Comment:
Since we're explicitly testing this, what are your thoughts on updating the validator with something like `name: Joi.forbidden()`?